src/lib/Composer/SubscriptionPlugin.php

```diff
@@ -0,0 +1,137 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
```

Comment:
seems to need an update (Ibexa), also php cs fixer might be out of date here as it did not catch this.